src/cli.rs

```diff
@@ -239,9 +237,7 @@ async fn get_deployment_data(
             .arg(format!("let r = import {}/.; in if builtins.isFunction r then (r {{}}).deploy else r.deploy", flake.repo))
     };
 
-    for extra_arg in extra_build_args {
-        c.arg(extra_arg);
-    }
+    c.args(extra_build_args);
```

Comment:
Can you also add `--remote-build` flag to the command line arguments of `deploy`?

Reply:
as a "build all deployments on the respective target servers" override?
the current implementation only builds selected builds remotely, but that could be a good idea

Reply:
> as a "build all deployments on the respective target servers" override?

Yes, I imagine that it might be useful when one doesn't have decent internet connection or has limited traffic